lib/e2e/Cargo.toml

```diff
@@ -10,13 +10,13 @@ repository.workspace = true
 
 [dependencies]
 alloy.workspace = true
-tokio = { version = "1.12.0", features = ["full", "process"] }
```

Comment:
Let's keep this one as well, since it uses both `features = ["full", "process"]` and set it to only use `features = ["full"]` in the workspace root.